d3-series/materials/Week 3/Assignment/Viz1.html

```diff
@@ -0,0 +1,21 @@
+<html>
+
+    <head>
+        <title>Viz1</title>
+    </head>
+
+    <body>
+        <svg id="viz" width='500' height='500'>
+                <rect x= "100" y="100" width="100" height="20" style="fill:rgb(0,0,255)"/>

```

Comment:
Ah okay the instructions for this first part may have been a little misleading, but this part also entailed using d3 to insert svg elements into the DOM by including the javascript in the script tags. What you have here is correct though!